### PR TITLE
Add rows attribute to prettytable

### LINKED ISSUE #128: PrettyTable should expose its data for consumption by other table libraries
### Is your feature request related to a problem? Please describe.

If I construct a PrettyTable, it's only useful for display... it's a dead end... if I want to then use it with other libraries (like pandas, matplotlib, openpyxl, etc), I have no way to access a PrettyTable's data via the public API.

### Describe the solution you'd like

I'd like if there was a `rows` or `data` attribute exposed publicly so that the table could be an input to downstream functions.

### Describe alternatives you've considered

Accessing the `table._rows` "hidden" attribute works fine, it just means I'm relying on a private implementation detail instead of a part of the API.

`df = pd.DataFrame(table._rows, columns=table.field_names)` works fine and is what I'm currently using.

### PR DESCRIPTION
**Pull request by kumaraditya303**
Closes https://github.com/jazzband/prettytable/issues/128

### CODE REVIEW

File: tests/test_prettytable.py [hunk 1]
```diff
@@ -344,6 +344,11 @@ def init_db(db_cursor):
 class TestBasic:
     """Some very basic tests."""
 
+    def test_table_rows(self, city_data_prettytable: PrettyTable) -> None:
+        rows = city_data_prettytable.rows[:]
```
Comment (hugovk) [inline tests/test_prettytable.py]: And can we skip making a copy here?
```suggestion
        rows = city_data_prettytable.rows
```

File: src/prettytable/prettytable.py [hunk 2]
```diff
@@ -518,6 +518,9 @@ def _validate_attributes(self, name, val):
     ##############################
     # ATTRIBUTE MANAGEMENT       #
     ##############################
+    @property
+    def rows(self):
+        return self._rows
```
Comment (hugovk) [inline src/prettytable/prettytable.py]: I think my earlier review got lost, take 2:

Can we return a copy here so the original is not writable?

```suggestion
        return self._rows[:]
```

**Comment by hugovk** [comment]
@kumaraditya303 It's fine this time, but please don't merge your own PRs without an approved review. Anyway, thank you for the contribution!

**Comment by kumaraditya303** [comment]
> @kumaraditya303 It's fine this time, but please don't merge your own PRs without an approved review. Anyway, thank you for the contribution!

Okay, I thought I addressed your suggestion so it was okay to merge myself.

**Comment by hugovk** [comment]
Yep, you did, although sometimes (usually with larger PRs) I will make partial suggestions and still want to do a final review of the whole thing, and maybe test locally, before merge.

**Pull request merged**